devtools-ci-index.yaml

```diff
@@ -623,6 +623,61 @@
         - archive:
             artifacts: ./perfcake-7.4/perfcake-chart
 
+- job:
+    name: 'devtools-test-end-to-end
+    defaults: global
+    node: devtools
+    properties:
+        - github:
+            url: https://github.com/fabric8io/fabric8-planner
+    scm:
+        - git:
+            url: https://github.com/fabric8io/fabric8-planner.git
+            shallow_clone: true
+            branches:
+                - master
+    triggers:
+        - timed: '30 08,12,16,20 * * *'
+    builders:
+        - shell: |
+            set +e
+            set +x
+            env > jenkins-env
+            cat $creds_config_file >> jenkins-env
+            export CICO_API_KEY=$(cat ~/duffy.key )
+            # get node
+            n=1
+            while true
+            do
+                cico_output=$(cico node get -f value -c ip_address -c comment)
+                if [ $? -eq 0 ]; then
+                    read CICO_hostname CICO_ssid <<< $cico_output
+                    if  [ ! -z "$CICO_hostname" ]; then
+                        # we got hostname from cico
+                        break
+                    fi
+                    echo "'cico node get' succeed, but can't get hostname from output"
+                fi
+                if [ $n -gt 5 ]; then
+                    # give up after 5 tries
+                    echo "giving up on 'cico node get'"
+                    exit 1
+                fi
+                echo "'cico node get' failed, trying again in 60s ($n/5)"
+                n=$[$n+1]
+                sleep 60
+            done
+            echo 'Using Host' $CICO_hostname
+            set -x
+            sshopts="-t -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -l root"
+            ssh_cmd="ssh $sshopts $CICO_hostname"
+            $ssh_cmd yum -y install rsync
+            rsync -e "ssh $sshopts" -Ha $(pwd)/ $CICO_hostname:payload
+            $ssh_cmd -t "cd payload && /bin/bash cico_run_EE_tests.sh $EE_TEST_USERNAME $EE_TEST_PASSWORD"
```

Comment:
where will it get these username and passwords from ? We would need those stored securely and also need to check the username/password are not being printed out to the console (i.e. https://ci.centos.org/view/Devtools/job/devtools-almighty-core/2721/console)

If we put them as is now they will end up in the console outpout afaics.